### ISSUE TITLE
This needs to be styled with branding for production 

### ISSUE THREAD

**Issue by 01101000011011110110010100001010** [Harems-io/web3-minting#1]
We would like to try a few different looks for this, which will allow you to use your best judgment with a little direction.

**Comment by nresh** [comment]
@01101000011011110110010100001010 I'm gonna need a little bit more specifications to deliver on this efficiently.  perhaps you can just send me someone else's similar page that you like and I can model something off of that?

and if you do have a similar site in mind, along with the url can you send any assets (e.g. particular images, logo, etc) that should live on this page?

**Comment by 01101000011011110110010100001010** [comment]
@nresh check discord and let me know if you need more info.

**Comment by nresh** [comment]
yep I saw it, I think that's enough to work with - I should have a PR for you tonight

**Comment by nresh** [comment]
@01101000011011110110010100001010 I have the first iteration of this ready on #2  

you can see live demo here: https://web3-minting.vercel.app/

lmk if you want any tweaks
